### ISSUE TITLE
Documentation build error: `could not convert string to float: 'background-color'`

### ISSUE THREAD

**Issue by namurphy** [PlasmaPy/PlasmaPy#2908]
We recently started getting a documentation build error on Read the Docs (see below).  It looks like it's in the `charged_particle_radiography_film_stacks` notebook.  I have a suspicion this error started happening because of a change in an upstream package.  

```python
reading sources... [ 92%] notebooks/diagnostics/charged_particle_radiography_film_stacks

====================== slowest reading durations =======================
16.291 notebooks/analysis/nullpoint
12.802 notebooks/analysis/fit_functions
4.858 notebooks/analysis/swept_langmuir/find_floating_potential
1.299 contributing/doc_guide
0.861 formulary/index

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbsphinx/__init__.py", line 634, in parse
    rststring, resources = exporter.from_notebook_node(nb, resources)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbsphinx/__init__.py", line 397, in from_notebook_node
    nb, resources = pp.preprocess(nb, resources)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbconvert/preprocessors/execute.py", line 103, in preprocess
    self.preprocess_cell(cell, resources, index)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbconvert/preprocessors/execute.py", line 124, in preprocess_cell
    cell = self.execute_cell(cell, index, store_history=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/.asdf/installs/python/3.12.3/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
arr = np.loadtxt(tissue_path, skiprows=8)
energy_axis = arr[:, 0] * u.MeV
tissue_density = 1.04 * u.g / u.cm**3
tissue_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * tissue_density

arr = np.loadtxt(aluminum_path, skiprows=8)
aluminum_density = 2.7 * u.g / u.cm**3
aluminum_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * aluminum_density
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;31mValueError[0m: could not convert string to float: 'background-color:'

The above exception was the direct cause of the following exception:

[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[4], line 1[0m
[0;32m----> 1[0m arr [38;5;241m=[39m [43mnp[49m[38;5;241;43m.[39;49m[43mloadtxt[49m[43m([49m[43mtissue_path[49m[43m,[49m[43m [49m[43mskiprows[49m[38;5;241;43m=[39;49m[38;5;241;43m8[39;49m[43m)[49m
[1;32m      2[0m energy_axis [38;5;241m=[39m arr[:, [38;5;241m0[39m] [38;5;241m*[39m u[38;5;241m.[39mMeV
[1;32m      3[0m tissue_density [38;5;241m=[39m [38;5;241m1.04[39m [38;5;241m*[39m u[38;5;241m.[39mg [38;5;241m/[39m u[38;5;241m.[39mcm[38;5;241m*[39m[38;5;241m*[39m[38;5;241m3[39m

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1397[0m, in [0;36mloadtxt[0;34m(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin, encoding, max_rows, quotechar, like)[0m
[1;32m   1394[0m [38;5;28;01mif[39;00m [38;5;28misinstance[39m(delimiter, [38;5;28mbytes[39m):
[1;32m   1395[0m     delimiter [38;5;241m=[39m delimiter[38;5;241m.[39mdecode([38;5;124m'[39m[38;5;124mlatin1[39m[38;5;124m'[39m)
[0;32m-> 1397[0m arr [38;5;241m=[39m [43m_read[49m[43m([49m[43mfname[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m
[1;32m   1398[0m [43m            [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiprows[49m[43m,[49m[43m [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m
[1;32m   1399[0m [43m            [49m[43munpack[49m[38;5;241;43m=[39;49m[43munpack[49m[43m,[49m[43m [49m[43mndmin[49m[38;5;241;43m=[39;49m[43mndmin[49m[43m,[49m[43m [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m
[1;32m   1400[0m [43m            [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquotechar[49m[43m)[49m
[1;32m   1402[0m [38;5;28;01mreturn[39;00m arr

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1036[0m, in [0;36m_read[0;34m(fname, delimiter, comment, quote, imaginary_unit, usecols, skiplines, max_rows, converters, ndmin, unpack, dtype, encoding)[0m
[1;32m   1033[0m     data [38;5;241m=[39m _preprocess_comments(data, comments, encoding)
[1;32m   1035[0m [38;5;28;01mif[39;00m read_dtype_via_object_chunks [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m-> 1036[0m     arr [38;5;241m=[39m [43m_load_from_filelike[49m[43m([49m
[1;32m   1037[0m [43m        [49m[43mdata[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquote[49m[43m,[49m
[1;32m   1038[0m [43m        [49m[43mimaginary_unit[49m[38;5;241;43m=[39;49m[43mimaginary_unit[49m[43m,[49m
[1;32m   1039[0m [43m        [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiplines[49m[43m,[49m[43m [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m
[1;32m   1040[0m [43m        [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m
[1;32m   1041[0m [43m        [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m[43m [49m[43mfilelike[49m[38;5;241;43m=[39;49m[43mfilelike[49m[43m,[49m
[1;32m   1042[0m [43m        [49m[43mbyte_converters[49m[38;5;241;43m=[39;49m[43mbyte_converters[49m[43m)[49m
[1;32m   1044[0m [38;5;28;01melse[39;00m:
[1;32m   1045[0m     [38;5;66;03m# This branch reads the file into chunks of object arrays and then[39;00m
[1;32m   1046[0m     [38;5;66;03m# casts them to the desired actual dtype.  This ensures correct[39;00m
[1;32m   1047[0m     [38;5;66;03m# string-length and datetime-unit discovery (like `arr.astype()`).[39;00m
[1;32m   1048[0m     [38;5;66;03m# Due to chunking, certain error reports are less clear, currently.[39;00m
[1;32m   1049[0m     [38;5;28;01mif[39;00m filelike:

[0;31mValueError[0m: could not convert string 'background-color:' to float64 at row 0, column 1.


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/cmd/build.py", line 337, in build_main
    app.build(args.force_all, args.filenames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/application.py", line 351, in build
    self.builder.build_update()
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 293, in build_update
    self.build(to_build,
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 313, in build
    updated_docnames = set(self.read())
                           ^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 419, in read
    self._read_serial(docnames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 440, in _read_serial
    self.read_doc(docname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/builders/__init__.py", line 497, in read_doc
    publisher.publish()
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/docutils/core.py", line 234, in publish
    self.document = self.reader.read(self.source, self.parser,
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/sphinx/io.py", line 107, in read
    self.parse()
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/docutils/readers/__init__.py", line 76, in parse
    self.parser.parse(self.input, document)
  File "/home/docs/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/nbsphinx/__init__.py", line 641, in parse
    raise NotebookError('\n'.join(lines))
nbsphinx.NotebookError: CellExecutionError in notebooks/diagnostics/charged_particle_radiography_film_stacks.ipynb:
------------------
arr = np.loadtxt(tissue_path, skiprows=8)
energy_axis = arr[:, 0] * u.MeV
tissue_density = 1.04 * u.g / u.cm**3
tissue_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * tissue_density

arr = np.loadtxt(aluminum_path, skiprows=8)
aluminum_density = 2.7 * u.g / u.cm**3
aluminum_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * aluminum_density
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;31mValueError[0m: could not convert string to float: 'background-color:'

The above exception was the direct cause of the following exception:

[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[4], line 1[0m
[0;32m----> 1[0m arr [38;5;241m=[39m [43mnp[49m[38;5;241;43m.[39;49m[43mloadtxt[49m[43m([49m[43mtissue_path[49m[43m,[49m[43m [49m[43mskiprows[49m[38;5;241;43m=[39;49m[38;5;241;43m8[39;49m[43m)[49m
[1;32m      2[0m energy_axis [38;5;241m=[39m arr[:, [38;5;241m0[39m] [38;5;241m*[39m u[38;5;241m.[39mMeV
[1;32m      3[0m tissue_density [38;5;241m=[39m [38;5;241m1.04[39m [38;5;241m*[39m u[38;5;241m.[39mg [38;5;241m/[39m u[38;5;241m.[39mcm[38;5;241m*[39m[38;5;241m*[39m[38;5;241m3[39m

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1397[0m, in [0;36mloadtxt[0;34m(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin, encoding, max_rows, quotechar, like)[0m
[1;32m   1394[0m [38;5;28;01mif[39;00m [38;5;28misinstance[39m(delimiter, [38;5;28mbytes[39m):
[1;32m   1395[0m     delimiter [38;5;241m=[39m delimiter[38;5;241m.[39mdecode([38;5;124m'[39m[38;5;124mlatin1[39m[38;5;124m'[39m)
[0;32m-> 1397[0m arr [38;5;241m=[39m [43m_read[49m[43m([49m[43mfname[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m
[1;32m   1398[0m [43m            [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiprows[49m[43m,[49m[43m [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m
[1;32m   1399[0m [43m            [49m[43munpack[49m[38;5;241;43m=[39;49m[43munpack[49m[43m,[49m[43m [49m[43mndmin[49m[38;5;241;43m=[39;49m[43mndmin[49m[43m,[49m[43m [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m
[1;32m   1400[0m [43m            [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquotechar[49m[43m)[49m
[1;32m   1402[0m [38;5;28;01mreturn[39;00m arr

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1036[0m, in [0;36m_read[0;34m(fname, delimiter, comment, quote, imaginary_unit, usecols, skiplines, max_rows, converters, ndmin, unpack, dtype, encoding)[0m
[1;32m   1033[0m     data [38;5;241m=[39m _preprocess_comments(data, comments, encoding)
[1;32m   1035[0m [38;5;28;01mif[39;00m read_dtype_via_object_chunks [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m-> 1036[0m     arr [38;5;241m=[39m [43m_load_from_filelike[49m[43m([49m
[1;32m   1037[0m [43m        [49m[43mdata[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquote[49m[43m,[49m
[1;32m   1038[0m [43m        [49m[43mimaginary_unit[49m[38;5;241;43m=[39;49m[43mimaginary_unit[49m[43m,[49m
[1;32m   1039[0m [43m        [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiplines[49m[43m,[49m[43m [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m
[1;32m   1040[0m [43m        [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m
[1;32m   1041[0m [43m        [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m[43m [49m[43mfilelike[49m[38;5;241;43m=[39;49m[43mfilelike[49m[43m,[49m
[1;32m   1042[0m [43m        [49m[43mbyte_converters[49m[38;5;241;43m=[39;49m[43mbyte_converters[49m[43m)[49m
[1;32m   1044[0m [38;5;28;01melse[39;00m:
[1;32m   1045[0m     [38;5;66;03m# This branch reads the file into chunks of object arrays and then[39;00m
[1;32m   1046[0m     [38;5;66;03m# casts them to the desired actual dtype.  This ensures correct[39;00m
[1;32m   1047[0m     [38;5;66;03m# string-length and datetime-unit discovery (like `arr.astype()`).[39;00m
[1;32m   1048[0m     [38;5;66;03m# Due to chunking, certain error reports are less clear, currently.[39;00m
[1;32m   1049[0m     [38;5;28;01mif[39;00m filelike:

[0;31mValueError[0m: could not convert string 'background-color:' to float64 at row 0, column 1.

You can ignore this error by setting the following in conf.py:

    nbsphinx_allow_errors = True


Notebook error:
CellExecutionError in notebooks/diagnostics/charged_particle_radiography_film_stacks.ipynb:
------------------
arr = np.loadtxt(tissue_path, skiprows=8)
energy_axis = arr[:, 0] * u.MeV
tissue_density = 1.04 * u.g / u.cm**3
tissue_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * tissue_density

arr = np.loadtxt(aluminum_path, skiprows=8)
aluminum_density = 2.7 * u.g / u.cm**3
aluminum_stopping_power = arr[:, 1] * u.MeV * u.cm**2 / u.g * aluminum_density
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;31mValueError[0m: could not convert string to float: 'background-color:'

The above exception was the direct cause of the following exception:

[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[4], line 1[0m
[0;32m----> 1[0m arr [38;5;241m=[39m [43mnp[49m[38;5;241;43m.[39;49m[43mloadtxt[49m[43m([49m[43mtissue_path[49m[43m,[49m[43m [49m[43mskiprows[49m[38;5;241;43m=[39;49m[38;5;241;43m8[39;49m[43m)[49m
[1;32m      2[0m energy_axis [38;5;241m=[39m arr[:, [38;5;241m0[39m] [38;5;241m*[39m u[38;5;241m.[39mMeV
[1;32m      3[0m tissue_density [38;5;241m=[39m [38;5;241m1.04[39m [38;5;241m*[39m u[38;5;241m.[39mg [38;5;241m/[39m u[38;5;241m.[39mcm[38;5;241m*[39m[38;5;241m*[39m[38;5;241m3[39m

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1397[0m, in [0;36mloadtxt[0;34m(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin, encoding, max_rows, quotechar, like)[0m
[1;32m   1394[0m [38;5;28;01mif[39;00m [38;5;28misinstance[39m(delimiter, [38;5;28mbytes[39m):
[1;32m   1395[0m     delimiter [38;5;241m=[39m delimiter[38;5;241m.[39mdecode([38;5;124m'[39m[38;5;124mlatin1[39m[38;5;124m'[39m)
[0;32m-> 1397[0m arr [38;5;241m=[39m [43m_read[49m[43m([49m[43mfname[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m
[1;32m   1398[0m [43m            [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiprows[49m[43m,[49m[43m [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m
[1;32m   1399[0m [43m            [49m[43munpack[49m[38;5;241;43m=[39;49m[43munpack[49m[43m,[49m[43m [49m[43mndmin[49m[38;5;241;43m=[39;49m[43mndmin[49m[43m,[49m[43m [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m
[1;32m   1400[0m [43m            [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquotechar[49m[43m)[49m
[1;32m   1402[0m [38;5;28;01mreturn[39;00m arr

File [0;32m~/checkouts/readthedocs.org/user_builds/plasmapy/envs/2906/lib/python3.12/site-packages/numpy/lib/_npyio_impl.py:1036[0m, in [0;36m_read[0;34m(fname, delimiter, comment, quote, imaginary_unit, usecols, skiplines, max_rows, converters, ndmin, unpack, dtype, encoding)[0m
[1;32m   1033[0m     data [38;5;241m=[39m _preprocess_comments(data, comments, encoding)
[1;32m   1035[0m [38;5;28;01mif[39;00m read_dtype_via_object_chunks [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m-> 1036[0m     arr [38;5;241m=[39m [43m_load_from_filelike[49m[43m([49m
[1;32m   1037[0m [43m        [49m[43mdata[49m[43m,[49m[43m [49m[43mdelimiter[49m[38;5;241;43m=[39;49m[43mdelimiter[49m[43m,[49m[43m [49m[43mcomment[49m[38;5;241;43m=[39;49m[43mcomment[49m[43m,[49m[43m [49m[43mquote[49m[38;5;241;43m=[39;49m[43mquote[49m[43m,[49m
[1;32m   1038[0m [43m        [49m[43mimaginary_unit[49m[38;5;241;43m=[39;49m[43mimaginary_unit[49m[43m,[49m
[1;32m   1039[0m [43m        [49m[43musecols[49m[38;5;241;43m=[39;49m[43musecols[49m[43m,[49m[43m [49m[43mskiplines[49m[38;5;241;43m=[39;49m[43mskiplines[49m[43m,[49m[43m [49m[43mmax_rows[49m[38;5;241;43m=[39;49m[43mmax_rows[49m[43m,[49m
[1;32m   1040[0m [43m        [49m[43mconverters[49m[38;5;241;43m=[39;49m[43mconverters[49m[43m,[49m[43m [49m[43mdtype[49m[38;5;241;43m=[39;49m[43mdtype[49m[43m,[49m
[1;32m   1041[0m [43m        [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m[43m [49m[43mfilelike[49m[38;5;241;43m=[39;49m[43mfilelike[49m[43m,[49m
[1;32m   1042[0m [43m        [49m[43mbyte_converters[49m[38;5;241;43m=[39;49m[43mbyte_converters[49m[43m)[49m
[1;32m   1044[0m [38;5;28;01melse[39;00m:
[1;32m   1045[0m     [38;5;66;03m# This branch reads the file into chunks of object arrays and then[39;00m
[1;32m   1046[0m     [38;5;66;03m# casts them to the desired actual dtype.  This ensures correct[39;00m
[1;32m   1047[0m     [38;5;66;03m# string-length and datetime-unit discovery (like `arr.astype()`).[39;00m
[1;32m   1048[0m     [38;5;66;03m# Due to chunking, certain error reports are less clear, currently.[39;00m
[1;32m   1049[0m     [38;5;28;01mif[39;00m filelike:

[0;31mValueError[0m: could not convert string 'background-color:' to float64 at row 0, column 1.

You can ignore this error by setting the following in conf.py:

    nbsphinx_allow_errors = True
```

**Comment by namurphy** [comment]
I'm testing this in #2909 to see if I can isolate which change caused the new doc build error.

Also, I'm reminded that CI errors that arise due to problems in upstream dependencies _sometimes_ go away when the developers due a bugpatch release, so sometimes it pays to procrastinate. 😅 On the other hand, this often requires the community to raise issues. 🌠

**Comment by pheuer** [comment]
I've been looking at this too on #2911 ... no idea what's going on. The text files being read definitely don't have the string 'background-color:' in them...

**Comment by namurphy** [comment]
My current suspicion is that it's due to a change in the Read the Docs software stack...but I haven't been able to track it down. I'm bogged down by proposal writing until Dec 2, but will take a deeper dive into this shortly thereafter.  Thanks!

**Comment by pheuer** [comment]
I've partially debugged this by adding logging statements: it looks like the Downloader module is getting an error page from GitHub instead of the file it should be getting: 

This is the output it receives. 
```
<!DOCTYPE html>
<html>
  <head>
    <meta content="origin" name="referrer">
    <title>Forbidden &middot; GitHub</title>
    <style type="text/css" media="screen">
      body {
        background-color: #f1f1f1;
        margin: 0;
      }
      body,
      input,
      button {
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      }
      .container { margin: 30px auto 40px auto; width: 800px; text-align: center; }
      a { color: #4183c4; text-decoration: none; font-weight: bold; }
      a:hover { text-decoration: underline; }
      h1, h2, h3 { color: #666; }
      ul { list-style: none; padding: 25px 0; }
      li {
        display: inline;
        margin: 10px 50px 10px 0px;
      }
      .logo { display: inline-block; margin-top: 35px; }
      .logo-img-2x { display: none; }
      @media
      only screen and (-webkit-min-device-pixel-ratio: 2),
      only screen and (   min--moz-device-pixel-ratio: 2),
      only screen and (     -o-min-device-pixel-ratio: 2/1),
      only screen and (        min-device-pixel-ratio: 2),
      only screen and (                min-resolution: 192dpi),
      only screen and (                min-resolution: 2dppx) {
        .logo-img-1x { display: none; }
        .logo-img-2x { display: inline-block; }
      }
    </style>
  </head>
  <body>

    <div class="container">
      <h1>Access to this site has been restricted.</h1>

      <p>
        <br>
        If you believe this is an error,
        please contact <a href="https://support.github.com">Support</a>.
      </p>

      <div id="s">
        <a href="https://githubstatus.com">GitHub Status</a> &mdash;
        <a href="https://twitter.com/githubstatus">@githubstatus</a>
      </div>
    </div>
  </body>
</html>
```

**Comment by pheuer** [comment]
@namurphy long term, this isn't the first time we've had issues accessing data files on the data repo. We could bundle these with PlasmaPy, but we already have a few Mb of data, and I hope we'll have more in the future. Is there any way we could host these files on the plasmapy.org domain?

**Comment by pheuer** [comment]
See fix in #2915 - the problem was that GitHub was classifying this as potentially malicious and blocking the request. It's fixed for now, but long term I still think it would be better to directly host an HTTP fileserver on the plasmapy.org domain.
